src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java

```diff
@@ -1948,13 +1948,8 @@ public void execute(EditingTarget target)
             // helper command to focus after navigation has completed
             final Command onNavigationCompleted = () ->
             {
-               if (file.focusOnNavigate())
-               {
-                  Scheduler.get().scheduleDeferred(() ->
-                  {
-                     target.focus();
-                  });
-               }
+               events_.fireEvent(new SuppressNextShellFocusEvent());
+               target.focus();
```

Comment:
I don't get this part of the change -- it looks like this was the only place where we read the `focusOnNavigate` flag. Is the flag still necessary?

Reply:
It is -- I removed that code unintentionally. What's the brain equivalent of a typo?